simple/stats/schema_constants.py

```diff
@@ -41,3 +41,7 @@
 
 CUSTOM_SVG_PREFIX = "c/g/"
 ROOT_SVG_ID = "dc/g/Root"
+
+# SV hierarchy groups will be put under this root.
+HIERARCHY_ROOT_SVG_ID = f"{CUSTOM_SVG_PREFIX}hierarchy"
+HIERARCHY_ROOT_SVG_NAME = "Hierarchy"
```

Comment:
Let me know if you can think of a better name for this.

Reply:
Variables?  Custom DC name?

Reply:
The name will be `"Custom Variables"` (that of the custom dc default root).